### DISCUSSION TITLE
Rack Group

### DISCUSSION THREAD

**Discussion by adeeb1981** [netbox-community/netbox#5599] [Ideas]
Hello

In Rack Groups, is it possible to customize to Rooms

Site --> Rooms --> Racks --> Devices

Add Floor Plan for Rooms (Rack Groups)

**Comment by cpmills1975** [comment]
A rack group can be a room if you like - there is no functionality to change the name "Rack Group" to anything else, but how you interpret it is up to you. In recent versions, you can nest rack groups, so you can (for example) model Site -> Rooms -> Rows -> Projects -> Racks -> Devices. Whatever works for you.

There is a FR open to allow devices to be assigned to a rack group - perhaps so a rack group can be used to model a store room or a cupboard. The functionality is flexible, but you will have to make do with the generic term "Rack Group".

Indeed, the flexibility is better than some other applications which have very strict concepts of rooms, aisles and rows. One application I use will only allow a very strict assignment of one to the other and the number of levels is limited unlike NetBox's more flexible concept of Rack Groups.
